modules/oadp-setting-resource-limits-and-requests.adoc

```diff
@@ -38,4 +38,9 @@ spec:
             memory: 256Mi
 ----
 <1> Specify the node selector to be supplied to Velero podSpec.
-<2> The `resourceAllocations` listed are for average usage.
\ No newline at end of file
+<2> The `resourceAllocations` listed are for average usage.
+
+[NOTE]
+====
+Kopia is an option in OADP 1.3 and later releases. You can use Kopia for both file system backups and data mover use cases. You must use Kopia for the Built-in DataMover. Kopia is more resource intensive than Restic, and you might need to adjust the CPU and memory requirements accordingly.
```

Comment:
You can use Kopia for file system backups, and it is the only option for data mover cases with the Built-in DataMover.

Reply:
and then the rest, ofcourse